docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -14,7 +14,15 @@ Note which version you actually have before starting.
 
 ### Update the application to v4.5.latest
 
-Run:
+To avoid deprecations when updating from an older PHP version to PHP 8.2 or 8.3, run the following commands:
```

Comment:
I'd describe this in a new `## v4.6.8` section at the bottom - I think it's the best place to mention it, because it's something Partners have to do if they started their project before 4.6.8 (but not after - this configuration will be included out of the box for 4.6.8 and higher)